packages/pages/src/components/link/link.tsx

```diff
@@ -23,15 +23,15 @@ interface LinkConfig
 /**
  * The shape of the data passed to {@link Link} when directly passing an HREF to the Link component.
  */
-interface HREFLinkProps extends LinkConfig {
+export interface HREFLinkProps extends LinkConfig {
```

Comment:
should these new export interfaces be marked with `@public` as well? and perhaps moved to `types.ts` in link/ so they can be exported through `index.ts`